packages/cli/src/commands/governance/propose.ts

```diff
@@ -23,6 +23,7 @@ export default class Propose extends BaseCommand {
     deposit: Flags.string({ required: true, description: 'Amount of Gold to attach to proposal' }),
     from: CustomFlags.address({ required: true, description: "Proposer's address" }),
     force: Flags.boolean({ description: 'Skip execution check', default: false }),
+    noinfo: Flags.boolean({ description: 'Skip printing the proposal info', default: false }),
```

Comment:
nit: to me `noInfo` would be nicer.

Reply:
This info is being loaded from BlockScout explorer, if the explorer is not available (eg in tests) whole command will fail.